src/riskMetrics/DynamicRisk.m

```diff
@@ -0,0 +1,72 @@
+function DynamicRisk = DynamicRisk(data)
+%DynamicRisk function that compute the dynamic risk based on current
+%glucose concentration and its rate-of-change, using hyperbolic tangent with
+%pre-defined parameters as amplification function
+%
+%Reference:
+%  - S. Guerra et al., "A Dynamic Risk Measure from Continuous Glucose
+%  Monitoring Data", Diabetes Technol. Ther., 2011. 
+%  doi: https://doi.org/10.1089/dia.2011.0006
+%
+% The function ignores nan values
```

Comment:
Put this in the "function scope comment". For an example, see https://github.com/gcappon/agata/blob/master/src/time/timeInHyperglycemia.m